src/h5web/providers/jupyter/api.ts

```diff
@@ -60,7 +54,7 @@ export class JupyterApi implements ProviderAPI {
     return data;
   }
 
-  private async fetchMetadata(path: string): Promise<JupyterMetaResponse> {
+  private async _fetchMetadata(path: string): Promise<JupyterMetaResponse> {
```

Comment:
Could be renamed `fetchEntityMetadata`

Reply:
FYI, I've renamed it back to `fetchMetadata`, but yeah, your suggestion is more explicit.